New Vertical Tab UX refactor

prakashsvmx opened the pull request:
Issue Description:
https://github.com/miniohq/engineering/issues/306

<summary>  How does it look ? some screens:  </summary>

<details> 
![user_details](https://user-images.githubusercontent.com/23444145/141106051-1f08701a-906d-4c24-8eb5-118cb8a8a627.png)

![Object_details_new](https://user-images.githubusercontent.com/23444145/141106057-a1dea237-8108-4962-89af-93e1b14f14ea.png)

![policy_details](https://user-images.githubusercontent.com/23444145/141106062-de02c5c4-d06f-4320-837c-0cf928bd5851.png)

![bucket_details](https://user-images.githubusercontent.com/23444145/141106064-486b4719-2191-4f05-9fc4-bc9c95583f7d.png)

![iam_policy_list](https://user-images.githubusercontent.com/23444145/141106070-0d55d26d-72b5-4b81-9d53-a33b8b81823b.png)

Mobile Preview:
![image](https://user-images.githubusercontent.com/23444145/141307821-c3643ac3-95f3-422e-8788-eff6f706b2b0.png)


</details>

Note:
Icons, and other listing screens will be updated in subsequent PR.

dvaldivia commented:
after rebase, the tests should pass @prakashsvmx

dvaldivia commented:
Do you think we can make the component expand to the bottom of the screen? to fill all this remaining space

<img width="1392" alt="Screen Shot 2021-11-10 at 4 58 06 PM" src="https://user-images.githubusercontent.com/18384552/141217789-6cbd858b-6561-417b-8183-fad7989d8998.png">

prakashsvmx commented:
@dvaldivia 
the tab content area is designed with 100% height that, it will shrink/grow based on the content.
for example: ( the summary section height is larger than available view port height)

![image](https://user-images.githubusercontent.com/23444145/141238791-ed073988-806a-4cc4-a979-4341e6a9d0d3.png)

prakashsvmx commented:
@dvaldivia @bexsoft @adfost @Alevsk @jinapurapu  - This is ready for review. PTAL

Impacted Screens:
- MinIO Console
  - Buckets -> New Bucket -> Manage
  - Buckets -> Browse -> Upload Object -> Object Details
  - Users -> User Details
  - Groups -> Group Details
  - IAM Policies -> Policy Details
- Operator Console
  -  Tenant details
  -  Storage

dvaldivia reviewed (approved):
tested. lgtm

bexsoft commented on portal-ui/src/screens/Console/Tenants/TenantDetails/VolumesSummary.tsx:
```diff
@@ -52,7 +52,7 @@ const styles = (theme: Theme) =>
     },
     tableWrapper: {
-      height: "calc(100vh - 267px)",
+      height: "450px",
```
This had a calc to make the container full height size

bexsoft commented on portal-ui/src/screens/Console/Users/ListUsers.tsx:
```diff
@@ -82,7 +83,7 @@ const styles = (theme: Theme) =>
     },
     twHeight: {
-      minHeight: 600,
+      minHeight: "450px",
```
```suggestion
      minHeight: 450,
```

bexsoft commented on portal-ui/src/screens/Console/Users/UserDetails.tsx:
```diff
@@ -49,16 +48,32 @@ import history from "../../../history";
+      border: "1px solid #EAEAEA",
+    },
+    tableHeight: {
+      height: "400px",
```
```suggestion
      height: 400,
```

Please review table sizes, as I see that these values are hardcoded heights.

bexsoft reviewed (changes requested):
There is an issue in collaved view, tabs container are not center aligned & there is a missing border at the bottom of the container
<img width="574" alt="Screen Shot 2021-11-11 at 15 06 56 copy" src="https://user-images.githubusercontent.com/33497058/141382557-090d4851-eac0-4dbd-92e3-09abda26e572.png">
:

adfost reviewed (dismissed):
Handle merge conflicts.

prakashsvmx commented:
@bexsoft  Thank you for the detailed review. I have addressed the comments. PTAL.

Quick summary:
1. https://github.com/minio/console/pull/1210#pullrequestreview-804219064 - this is due to the short text. if the text is few characters long, it looks fine.
2. table height has been removed. ( calc is not a viable solution as we have tables at different levels). so iit has been left to the defaults of the `TableWrapper`.

dvaldivia reviewed (approved):
Tested. LGTM.

bexsoft reviewed (approved):
LGTM
